e2e/portal/tests/ProjectSettings/FSP configuration/ReconfigureFspConfiguration.spec.ts

```diff
@@ -0,0 +1,83 @@
+import test from '@playwright/test';
+
+import { SeedScript } from '@121-service/src/scripts/enum/seed-script.enum';
+import {
+  getAccessToken,
+  resetDB,
+} from '@121-service/test/helpers/utility.helper';
+
+import FspSettingsPage from '@121-e2e/portal/pages/FspSettingsPage';
+import HomePage from '@121-e2e/portal/pages/HomePage';
+import LoginPage from '@121-e2e/portal/pages/LoginPage';
+import RegistrationsPage from '@121-e2e/portal/pages/RegistrationsPage';
+
+const configuredFsps = ['Visa debit card', 'Albert Heijn voucher WhatsApp'];
+const visaConfigurationCi = [
+  'Visa debit card',
+  'test-INTERSOLVE_VISA_BRAND_CODE',
+  'TESTINTERSOLVEVISACOVERLETTERCODE',
+  'test_INTERSOLVE_VISA_FUNDINGTOKEN_CODE',
+];
+const visaConfigurationLocal = [
+  'Visa debit card',
+  'Ix906_01',
+  'RC01',
+  '6375100999151000001',
+];
```

Comment:
this doesn't seem great. Both the split in Ci/local, and the hardcoding. Is this necessary? (maybe I don't get it, so please explain)

Reply:
see other comment